### ISSUE TITLE
Crashes when hitting the load key

### ISSUE THREAD

**Issue by GubsHub** [Lunatrius/Schematica#203]
```
---- Minecraft Crash Report ----

WARNING: coremods are present:
Contact their authors BEFORE contacting forge

// There are four lights!

Time: 2/23/17 5:15 PM
Description: Unexpected error

java.lang.NoSuchMethodError: net.minecraft.item.ItemStack.func_77963_c(Lnet/minecraft/nbt/NBTTagCompound;)V
	at com.github.lunatrius.schematica.world.schematic.SchematicUtil.getIconFromNBT(SchematicUtil.java:69)
	at com.github.lunatrius.schematica.world.schematic.SchematicUtil.getIconFromFile(SchematicUtil.java:81)
	at com.github.lunatrius.schematica.client.gui.load.GuiSchematicLoad.reloadSchematics(GuiSchematicLoad.java:162)
	at com.github.lunatrius.schematica.client.gui.load.GuiSchematicLoad.func_73866_w_(GuiSchematicLoad.java:58)
	at net.minecraft.client.gui.GuiScreen.func_146280_a(GuiScreen.java:487)
	at net.minecraft.client.Minecraft.func_147108_a(Minecraft.java:966)
	at com.github.lunatrius.schematica.handler.client.InputHandler.onKeyInput(InputHandler.java:60)
	at net.minecraftforge.fml.common.eventhandler.ASMEventHandler_12_InputHandler_onKeyInput_InputEvent.invoke(.dynamic)
	at net.minecraftforge.fml.common.eventhandler.ASMEventHandler.invoke(ASMEventHandler.java:90)
	at net.minecraftforge.fml.common.eventhandler.EventBus.post(EventBus.java:185)
	at net.minecraftforge.fml.common.FMLCommonHandler.fireKeyInput(FMLCommonHandler.java:558)
	at net.minecraft.client.Minecraft.func_184118_az(Minecraft.java:1953)
	at net.minecraft.client.Minecraft.func_71407_l(Minecraft.java:1744)
	at net.minecraft.client.Minecraft.func_71411_J(Minecraft.java:1056)
	at net.minecraft.client.Minecraft.func_99999_d(Minecraft.java:372)
	at net.minecraft.client.main.Main.main(SourceFile:124)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:483)
	at net.minecraft.launchwrapper.Launch.launch(Launch.java:135)
	at net.minecraft.launchwrapper.Launch.main(Launch.java:28)


A detailed walkthrough of the error, its code path and all known details is as follows:
---------------------------------------------------------------------------------------

-- Head --
Thread: Client thread
Stacktrace:
	at com.github.lunatrius.schematica.world.schematic.SchematicUtil.getIconFromNBT(SchematicUtil.java:69)
	at com.github.lunatrius.schematica.world.schematic.SchematicUtil.getIconFromFile(SchematicUtil.java:81)
	at com.github.lunatrius.schematica.client.gui.load.GuiSchematicLoad.reloadSchematics(GuiSchematicLoad.java:162)
	at com.github.lunatrius.schematica.client.gui.load.GuiSchematicLoad.func_73866_w_(GuiSchematicLoad.java:58)
	at net.minecraft.client.gui.GuiScreen.func_146280_a(GuiScreen.java:487)
	at net.minecraft.client.Minecraft.func_147108_a(Minecraft.java:966)
	at com.github.lunatrius.schematica.handler.client.InputHandler.onKeyInput(InputHandler.java:60)
	at net.minecraftforge.fml.common.eventhandler.ASMEventHandler_12_InputHandler_onKeyInput_InputEvent.invoke(.dynamic)
	at net.minecraftforge.fml.common.eventhandler.ASMEventHandler.invoke(ASMEventHandler.java:90)
	at net.minecraftforge.fml.common.eventhandler.EventBus.post(EventBus.java:185)
	at net.minecraftforge.fml.common.FMLCommonHandler.fireKeyInput(FMLCommonHandler.java:558)
	at net.minecraft.client.Minecraft.func_184118_az(Minecraft.java:1953)

-- Affected level --
Details:
	Level name: MpServer
	All players: 1 total; [EntityPlayerSP['Zur'/116, l='MpServer', x=-854.37, y=29.36, z=-1695.60]]
	Chunk stats: MultiplayerChunkCache: 1088, 1088
	Level seed: 0
	Level generator: ID 01 - flat, ver 0. Features enabled: false
	Level generator options: 
	Level spawn location: World: (-853,4,-1739), Chunk: (at 11,0,5 in -54,-109; contains blocks -864,0,-1744 to -849,255,-1729), Region: (-2,-4; contains chunks -64,-128 to -33,-97, blocks -1024,0,-2048 to -513,255,-1537)
	Level time: 32595 game time, 4964 day time
	Level dimension: 0
	Level storage version: 0x00000 - Unknown?
	Level weather: Rain time: 0 (now: false), thunder time: 0 (now: false)
	Level game mode: Game mode: creative (ID 1). Hardcore: false. Cheats: false
	Forced entities: 46 total; [EntityBat['Bat'/64, l='MpServer', x=-855.25, y=8.10, z=-1665.25], EntityBat['Bat'/65, l='MpServer', x=-855.25, y=8.10, z=-1674.68], EntityBat['Bat'/66, l='MpServer', x=-860.56, y=8.10, z=-1666.42], EntityBat['Bat'/67, l='MpServer', x=-852.25, y=8.10, z=-1668.25], EntityBat['Bat'/68, l='MpServer', x=-855.25, y=8.10, z=-1665.25], EntityBat['Bat'/69, l='MpServer', x=-862.57, y=8.10, z=-1667.25], EntityBat['Bat'/70, l='MpServer', x=-853.18, y=8.10, z=-1668.52], EntityBat['Bat'/71, l='MpServer', x=-852.00, y=8.10, z=-1670.53], EntityBat['Bat'/72, l='MpServer', x=-860.57, y=7.97, z=-1665.25], EntityPig['Pig'/75, l='MpServer', x=-846.11, y=4.00, z=-1769.63], EntityPig['Pig'/76, l='MpServer', x=-841.71, y=4.00, z=-1763.38], EntityHorse['Horse'/77, l='MpServer', x=-838.20, y=4.00, z=-1758.20], EntityPig['Pig'/78, l='MpServer', x=-837.62, y=4.00, z=-1617.23], EntityVillager['Villager'/79, l='MpServer', x=-838.51, y=4.00, z=-1629.56], EntityHorse['Horse'/21, l='MpServer', x=-933.52, y=4.00, z=-1710.81], EntityPig['Pig'/86, l='MpServer', x=-832.48, y=4.00, z=-1637.72], EntityCow['Cow'/25, l='MpServer', x=-914.30, y=4.00, z=-1749.21], EntitySheep['Sheep'/26, l='MpServer', x=-918.81, y=4.00, z=-1749.67], EntitySheep['Sheep'/27, l='MpServer', x=-917.05, y=4.00, z=-1743.99], EntitySheep['Sheep'/28, l='MpServer', x=-917.76, y=4.00, z=-1714.74], EntitySheep['Sheep'/29, l='MpServer', x=-915.35, y=4.00, z=-1724.30], EntityCow['Cow'/30, l='MpServer', x=-920.42, y=4.00, z=-1719.18], EntitySheep['Sheep'/31, l='MpServer', x=-914.23, y=4.00, z=-1724.20], EntityChicken['Chicken'/33, l='MpServer', x=-896.50, y=4.00, z=-1775.50], EntitySheep['Sheep'/34, l='MpServer', x=-910.16, y=4.00, z=-1755.16], EntitySheep['Sheep'/35, l='MpServer', x=-908.09, y=4.00, z=-1725.48], EntityCow['Cow'/36, l='MpServer', x=-900.76, y=4.00, z=-1723.30], EntityCow['Cow'/37, l='MpServer', x=-897.08, y=4.00, z=-1723.34], EntityPig['Pig'/38, l='MpServer', x=-905.72, y=4.00, z=-1653.80], EntitySheep['Sheep'/39, l='MpServer', x=-909.11, y=4.00, z=-1636.47], EntityChicken['Chicken'/42, l='MpServer', x=-890.50, y=4.00, z=-1774.50], EntityPlayerSP['Zur'/116, l='MpServer', x=-854.37, y=29.36, z=-1695.60], EntityCow['Cow'/43, l='MpServer', x=-894.91, y=4.00, z=-1725.97], EntityBat['Bat'/51, l='MpServer', x=-864.75, y=8.10, z=-1672.75], EntityPig['Pig'/52, l='MpServer', x=-865.35, y=4.00, z=-1616.11], EntityPig['Pig'/53, l='MpServer', x=-856.17, y=4.00, z=-1772.24], EntityPig['Pig'/54, l='MpServer', x=-853.50, y=4.00, z=-1765.50], EntitySheep['Sheep'/55, l='MpServer', x=-857.17, y=4.00, z=-1735.68], EntityBat['Bat'/56, l='MpServer', x=-856.25, y=8.10, z=-1684.75], EntityBat['Bat'/57, l='MpServer', x=-861.75, y=8.10, z=-1665.25], EntityBat['Bat'/58, l='MpServer', x=-861.82, y=8.10, z=-1670.19], EntityBat['Bat'/59, l='MpServer', x=-852.25, y=8.10, z=-1671.39], EntityBat['Bat'/60, l='MpServer', x=-852.32, y=8.10, z=-1668.25], EntityBat['Bat'/61, l='MpServer', x=-858.74, y=8.10, z=-1668.74], EntityBat['Bat'/62, l='MpServer', x=-856.25, y=8.10, z=-1679.61], EntityBat['Bat'/63, l='MpServer', x=-856.25, y=8.10, z=-1677.65]]
	Retry entities: 0 total; []
	Server brand: fml,forge
	Server type: Integrated singleplayer server
Stacktrace:
	at net.minecraft.client.multiplayer.WorldClient.func_72914_a(WorldClient.java:415)
	at net.minecraft.client.Minecraft.func_71396_d(Minecraft.java:2655)
	at net.minecraft.client.Minecraft.func_99999_d(Minecraft.java:401)
	at net.minecraft.client.main.Main.main(SourceFile:124)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:483)
	at net.minecraft.launchwrapper.Launch.launch(Launch.java:135)
	at net.minecraft.launchwrapper.Launch.main(Launch.java:28)

-- System Details --
Details:
	Minecraft Version: 1.11.2
	Operating System: Windows 10 (amd64) version 10.0
	Java Version: 1.8.0_25, Oracle Corporation
	Java VM Version: Java HotSpot(TM) 64-Bit Server VM (mixed mode), Oracle Corporation
	Memory: 312347472 bytes (297 MB) / 619446272 bytes (590 MB) up to 1060372480 bytes (1011 MB)
	JVM Flags: 6 total; -XX:HeapDumpPath=MojangTricksIntelDriversForPerformance_javaw.exe_minecraft.exe.heapdump -Xmx1G -XX:+UseConcMarkSweepGC -XX:+CMSIncrementalMode -XX:-UseAdaptiveSizePolicy -Xmn128M
	IntCache: cache: 0, tcache: 0, allocated: 0, tallocated: 0
	FML: MCP 9.38 Powered by Forge 13.20.0.2232 7 mods loaded, 7 mods active
	States: 'U' = Unloaded 'L' = Loaded 'C' = Constructed 'H' = Pre-initialized 'I' = Initialized 'J' = Post-initialized 'A' = Available 'D' = Disabled 'E' = Errored
	UCHIJAAAA	minecraft{1.11.2} [Minecraft] (minecraft.jar) 
	UCHIJAAAA	mcp{9.19} [Minecraft Coder Pack] (minecraft.jar) 
	UCHIJAAAA	FML{8.0.99.99} [Forge Mod Loader] (forge-1.11.2-13.20.0.2232.jar) 
	UCHIJAAAA	forge{13.20.0.2232} [Minecraft Forge] (forge-1.11.2-13.20.0.2232.jar) 
	UCHIJAAAA	mercurius_updater{1.0} [mercurius_updater] (MercuriusUpdater-1.11.jar) 
	UCHIJAAAA	lunatriuscore{1.1.2.39} [LunatriusCore] (LunatriusCore-1.11-1.1.2.39-universal (2).jar) 
	UCHIJAAAA	schematica{1.7.8.158} [Schematica] (Schematica-1.11-1.7.8.158-universal.jar) 
	Loaded coremods (and transformers): 
	GL info: ' Vendor: 'NVIDIA Corporation' Version: '4.5.0 NVIDIA 376.53' Renderer: 'GeForce GTX 1060 6GB/PCIe/SSE2'
	Launched Version: 1.11.2
	LWJGL: 2.9.4
	OpenGL: GeForce GTX 1060 6GB/PCIe/SSE2 GL version 4.5.0 NVIDIA 376.53, NVIDIA Corporation
	GL Caps: Using GL 1.3 multitexturing.
Using GL 1.3 texture combiners.
Using framebuffer objects because OpenGL 3.0 is supported and separate blending is supported.
Shaders are available because OpenGL 2.1 is supported.
VBOs are available because OpenGL 1.5 is supported.

	Using VBOs: Yes
	Is Modded: Definitely; Client brand changed to 'fml,forge'
	Type: Client (map_client.txt)
	Resource Packs: 
	Current Language: English (US)
	Profiler Position: N/A (disabled)
	CPU: 8x AMD FX(tm)-8320 Eight-Core Processor ```

**Comment by Lunatrius** [comment]
#192 

I'd suggest a clean install of Forge. `fml-client-latest.log` would help a lot if that doesn't fix it.

**Comment by GubsHub** [comment]
ty
